### PR TITLE
fix(LightRegularGrid): Added noexcept on ravlue constructors for LightRegularGrids.

### PR DESCRIPTION
**Pull request by MelchiorSchuh**

### CODE REVIEW

**Review by panquez — APPROVED** [review]

File: src/geode/mesh/core/coordinate_reference_system_managers.cpp [hunk 1]
```diff
@@ -154,15 +154,12 @@ namespace geode
...
-        : impl_{ std::move( other.impl_ ) }
-    {
-    }
+            CoordinateReferenceSystemManagers&& other ) noexcept = default;
 
     template < index_t dimension >
     CoordinateReferenceSystemManagers< dimension >&
         CoordinateReferenceSystemManagers< dimension >::operator=(
-            CoordinateReferenceSystemManagers&& other )
+            CoordinateReferenceSystemManagers&& other ) noexcept
```
Comment (BotellaA) [inline src/geode/mesh/core/coordinate_reference_system_managers.cpp:162]: = default?

File: src/geode/mesh/core/grid.cpp [hunk 2]
```diff
@@ -463,14 +463,10 @@ namespace geode
...
-    Grid< dimension >::Grid( Grid&& other )
-        : CellArray< dimension >( std::move( other ) ),
-          impl_( std::move( other.impl_ ) )
-    {
-    }
+    Grid< dimension >::Grid( Grid&& other ) noexcept = default;
 
     template < index_t dimension >
-    Grid< dimension >& Grid< dimension >::operator=( Grid&& other )
+    Grid< dimension >& Grid< dimension >::operator=( Grid&& other ) noexcept
```
Comment (BotellaA) [inline src/geode/mesh/core/grid.cpp:469]: = default?

File: src/geode/mesh/core/light_regular_grid.cpp [hunk 3]
```diff
@@ -86,12 +88,8 @@ namespace geode
...
-        : Grid< dimension >( std::move( other ) ),
-          Identifier( std::move( other ) ),
-          impl_( std::move( other.impl_ ) )
-    {
-    }
+    LightRegularGrid< dimension >::LightRegularGrid(
+        LightRegularGrid&& other ) noexcept = default;
 
     template < index_t dimension >
     LightRegularGrid< dimension >::~LightRegularGrid() // NOLINT
```
Comment (BotellaA) [inline src/geode/mesh/core/light_regular_grid.cpp:95]: = default

File: src/geode/mesh/core/solid_mesh.cpp [hunk 4]
```diff
@@ -608,16 +608,11 @@ namespace geode
...
-          CoordinateReferenceSystemManagers< dimension >( std::move( other ) ),
-          impl_( std::move( other.impl_ ) )
-    {
-    }
+    SolidMesh< dimension >::SolidMesh( SolidMesh&& other ) noexcept = default;
 
     template < index_t dimension >
     SolidMesh< dimension >& SolidMesh< dimension >::operator=(
-        SolidMesh&& other )
+        SolidMesh&& other ) noexcept
```
Comment (BotellaA) [inline src/geode/mesh/core/solid_mesh.cpp:615]: idem partout :p

**Review by BotellaA — APPROVED** [review]
Some changes if you can

**Pull request merged**